Packs/GoogleKeyManagementService/Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py

```diff
@@ -352,7 +351,7 @@ def get_update_command_body(args: Dict[str, Any], update_mask: List) -> Dict:
 
     if 'purpose' in update_mask:
         # Add purpose enum to body
-        body['purpose'] = kms.CryptoKey.CryptoKeyPurpose[args.get('purpose')].value
+        body['purpose'] = kms.CryptoKey.CryptoKeyPurpose[args['purpose']].value  # type: ignore[misc]
```

Comment:
get -> [] 
not sure we want to do this change

Reply:
```
if 'purpose' in update_mask:
```